src/xrCore/Threading/TaskManager.cpp

```diff
@@ -37,6 +37,8 @@ xr_unique_ptr<TaskManager> TaskScheduler;
 
 static constexpr size_t OTHER_THREADS_COUNT = 1; // Primary thread
 
+static thread_local CRandom random{};
```

Comment:
It's better to still have this variable in the TaskManager class.

Reply:
```c++
class XRCORE_API TaskManager final
{
private:
    ...
    static thread_local CRandom random;
```

```
"private: static CRandom TaskManager::random": данные с длительностью хранилища потока не могут иметь интерфейс dll
```

Reply:
I think it can be moved to s_tl_worker